polymesh-docs/primitives/identity.mdx

```diff
@@ -76,43 +76,72 @@ Polymesh uses public-key cryptography based on the [Ed25519 curve](https://docs.
+- have its *Paying Identity* set to the callers identity
+- issue `AddMultiSigSigner` authorisations to each of the specified signers
+
+Once created, the specified signers will need to accept the authorisations issued above, before they can interact with the multisig. This is done via a call to `multisig::acceptMultisigSigner` by each of the signing keys.
+
+#### Submitting Transactions
+
+MultiSig Signers are not directly associated with an identity, but are indirectly associated through their linked MultiSig (which may be attached to an identity as a primary or secondary key).
 
-All transactions have a single payer, which is established when the transaction is first submitted. For a transaction executed via a MultiSig this will be the Creator Identity derived from the external MultiSig Signer account that submits the extrinsic.
+This has the consequence that MultiSig Signers can not send / receive POLYX (since they are not directly associated with an identity), and hence cannot directly pay for transactions to create or approve proposals.
 
-When Signers are approving / proposing a MultiSig transaction, we have:
+In order to submit a normal transaction (excluding transactions related to joining / leaving an identity - for these see below), the MultiSig must be attached to a valid identity as a primary or secondary key, and either the MultiSig itself, or the primary key of its *Paying Identity*, must have sufficient funds to pay the transaction fees to create and approve the proposal.
 
-- CDD check done on Creator Identity
+#### Identity Management
 
-- Payment made by Creator Identity
+When initially created, a MultiSig is:
+- linked to its creating identity as a secondary key
+- has its *Paying Identity* set to the creating identity
+- has no *Admin Identity* assigned
 
-When a MultiSig executes a proposal, we have:
+However, after creation, these relationships can be amended.
 
-- CDD check done on Creator Identity
+In particular an *Admin Identity* can be assigned via `multisig::addAdmin`, which then allows the assigned *Admin Identity* to add / remove signers from the MultiSig (via `multisig::addMultisigSignersViaAdmin`, `multisig::removeMultisigSignersViaAdmin`) and change the number of approvals needed (via `multisig::changeSigsRequiredViaAdmin`). To make these changes, the primary key of the *Admin Identity* must call these transactions.
+
+Additionally a MultiSig can leave and join new identities, similar to any other key. Changing identity will not update the *Admin Identity* of the MultiSig - this must be amended separately if needed.
+
+To change identity, a MultiSig must:
+- leave its current identity by executing a proposal of `identity::leaveIdentityAsKey`
+- receive a `JoinIdentity` authorisation from another identity
+- create / approve the `JoinIdentity` authorisation by a direct call to `multisig::approveJoinIdentity` by the required number of signers
```

Comment:
It is also possible to batch the `leave_did` and `join_identity` calls and do this as a single atomic proposal.
The `multisig.approve_join_identity` is only needed if the MS as been removed from the DID.